### PR TITLE
Change 'consider to give name to results'

### PR DESCRIPTION
**Pull request by barbeque**
Looks like a great project! I noticed this warning was a little strangely written, so I decided to stand up this PR to fix it.

Thanks for your time.

### CODE REVIEW

File: lint/unnamedResult_checker.go [hunk 1]
```diff
@@ -82,5 +82,5 @@ func (c *unnamedResultChecker) typeName(typ types.Type) string {
 }
 
 func (c *unnamedResultChecker) warn(n ast.Node) {
-	c.ctx.Warn(n, "consider to give name to results")
+	c.ctx.Warn(n, "consider giving a name to this result")
```
Comment (cristaloleg) [inline lint/unnamedResult_checker.go]: `results` stands for `resulting parameters`.
Maybe we should leave a multiple form, so: `this results`.
Am I right?

Comment (barbeque) [reply lint/unnamedResult_checker.go]: `these results` perhaps?

Comment (cristaloleg) [reply lint/unnamedResult_checker.go]: Oh, yes 😉

**Comment by cristaloleg** [comment]
Will leave it open for a short time, so @Quasilyte will have a look at it. Thank you for your time 🙌

**Comment by Quasilyte** [comment]
Just a note: we're using golang/go conventions for commit messages.
https://golang.org/doc/contribute.html#commit_messages
I've squashed with updated message, so it fits these conventions.

**Pull request merged**